src/ripple/protocol/TxFormats.h

```diff
@@ -58,6 +58,7 @@ enum TxType {
 
     ttAMENDMENT = 100,
     ttFEE = 101,
+    ttUNL_MODIDY = 102,
```

Comment:
Typo (in multiple places): `ttUNL_MODIFY`

Reply:
Thanks. Changed.